### DISCUSSION TITLE
Pod origin yaml file path in the running Pod metatdata

### DISCUSSION THREAD

**Discussion by dzintars** [containers/podman#9815] [Ideas]
Today came across case where i have many `pod-*.yml` files and the running pod. And i have no idea by which `yaml` file that Pod was launched. As i see those pod files i'm sure `play kube` was used.
So... i would like to see some kind `origin config path` in the `podman pod inspect *` metadata.
Sure it's not the case for CI/CD pipelines, but for some manual development things it could be helpful.
I ended up with diffing generated pod with the existing ones.

**Comment by mheon** [comment]
Easiest way would be to add a label - io.podman.kube.source=<path> or
similar
